fdbserver/LogSystemConfig.h

```diff
@@ -139,14 +153,17 @@ struct TLogSet {
 	void serialize( Ar& ar ) {
 		if constexpr (is_fb_function<Ar>) {
 			serializer(ar, tLogs, logRouters, tLogWriteAntiQuorum, tLogReplicationFactor, tLogPolicy, tLogLocalities,
-			           isLocal, locality, startVersion, satelliteTagLocations, tLogVersion);
+			           isLocal, locality, startVersion, satelliteTagLocations, tLogVersion, backupWorkers);
 		} else {
 			serializer(ar, tLogs, logRouters, tLogWriteAntiQuorum, tLogReplicationFactor, tLogPolicy, tLogLocalities, isLocal, locality, startVersion, satelliteTagLocations);
 			if (ar.isDeserializing && !ar.protocolVersion().hasTLogVersion()) {
 				tLogVersion = TLogVersion::V2;
 			} else {
 				serializer(ar, tLogVersion);
 			}
+			if (ar.protocolVersion().hasBackupWorker()) {
```

Comment:
TLogSet is never persisted, so we do not need versioning on it. Delete everything besides the single serializer call